torchtyping/tensor_type.py

```diff
@@ -86,7 +88,11 @@ def __class_getitem__(cls, item: Any) -> _AnnotatedType:
         layouts = []
         details = []
         for item_i in item:
-            if isinstance(item_i, (int, str, slice)) or item_i in (None, ...):
+            if (
+                isinstance(item_i, (int, str, slice))
+                or item_i in (None, ...)
+                or item_i is Any
```

Comment:
These two lines can be simplified slightly to `item_i in (None, ..., Any)`

Reply:
You're right! I will correct it in the next commit :smile: